opentelemetry-operator.yaml

```diff
@@ -0,0 +1,93 @@
+          packages: ./cmd/otel-allocator
+          output: main
+          ldflags: |
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.version=$(grep -v '\#' versions.txt | grep operator= | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.buildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.otelCol=$(grep -v '\#' versions.txt | grep opentelemetry-collector | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.targetAllocator=$(grep -v '\#' versions.txt | grep targetallocator | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.operatorOpAMPBridge=$(grep -v '\#' versions.txt | grep operator-opamp-bridge | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationJava=$(grep -v '\#' versions.txt | grep autoinstrumentation-java | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationNodeJS=$(grep -v '\#' versions.txt | grep autoinstrumentation-nodejs | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationPython=$(grep -v '\#' versions.txt | grep autoinstrumentation-python | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationDotNet=$(grep -v '\#' versions.txt | grep autoinstrumentation-dotnet | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationGo=$(grep -v '\#' versions.txt | grep autoinstrumentation-go | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationApacheHttpd=$(grep -v '\#' versions.txt | grep autoinstrumentation-apache-httpd | awk -F= '{print $2}')
+            -X github.com/open-telemetry/opentelemetry-operator/internal/version.autoInstrumentationNginx=$(grep -v '\#' versions.txt | grep autoinstrumentation-nginx | awk -F= '{print $2}')
+    test:
+      pipeline:
+        - runs: |
+            main --help || true # test tends to fail
+
+  - name: ${{package.name}}-otel-allocator-compat
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          # The helm chart expects the binaries to be in / instead of /usr/bin
+          mkdir -p "${{targets.contextdir}}"
+          ln -sf /usr/bin/main ${{targets.contextdir}}/main
+
+update:
+  enabled: true
+  github:
+    identifier: open-telemetry/opentelemetry-operator
+    strip-prefix: v
+
+test:
+  pipeline:
+    # AUTOGENERATED
+    - runs: |
+        manager -h || true # test tends to fail
```

Comment:
Can we test the manager starts with kwok at least?

Reply:
let me dig into a bit on whether its doable

Reply:
done @maxgio92 would you mind taking a look? Thank you!